### ISSUE TITLE
Feature Request: Resubmit only translation

### ISSUE THREAD

**Issue by rDarge** [K-RT-Dev/VGT#6]
Occasionally OpenAI's API will be down - when this happens if I submit a new clip to be translated it just returns "Error". I'd like to be able to hit a button to resubmit just the last step of the translation process to the API, rather than have to make a new sample and OCR again. 

Something like the following: 
[output.webm](https://github.com/K-RT-Dev/VGT/assets/1133811/b53954ba-1f63-4e9d-8fdf-025cfd701394)

You mentioned you would be overhauling the options for translation services here, so maybe you've already done something like this; otherwise I think it would be a good addition even across different translation services.

**Comment by rDarge** [comment]
I've got a prototype working in my fork here: https://github.com/rDarge/VGT/commit/edd43921bcb14aada02f4bcb9296689fba75e9ea

If this sounds good to you, I can push it into an isolated branch and get it ready for PR w/ any feedback.

**Comment by K-RT-Dev** [comment]
Yes! This is something that was missing. Please send it to a branch. I think it will be necessary to modify it when the system with multiple translation engines is ready, but as a foundation, it's great.

**Comment by rDarge** [comment]
Cool, stay tuned for that - I should have something ready to go by the end of the week.
